tests/acceptance/30_custom_promise_types/22_what_module_gets.cf.expected

```diff
@@ -4,15 +4,15 @@ operation=validate_promise
 log_level=info
 promise_type=example
 promiser=Promiser
-line_number=44
+line_number=46
```

Comment:
and this change passes on other platforms? Maybe run this through jenkins?

Reply:
Yep, exactly! By adding two lines to `bundle agent test` `meta:` section of `22_what_module_gets.cf` file[^1], we move all following lines two lines lower.

[^1]: these lines are: `"test_soft_fail" string => "windows",` `meta => { "ENT-10217" };`